app.py

```diff
@@ -294,7 +295,9 @@ def load_chat_screen(assistant_id, assistant_title):
 
 def main():
     # Check if multi-agent settings are defined
-    multi_agents = os.environ.get("OPENAI_ASSISTANTS", None)
+    # multi_agents = os.environ.get("OPENAI_ASSISTANTS", None)
+    # load current from assistants.py
+    multi_agents = current_assistants()
```

Comment:
what are the actual benefits?

Reply:
why there is a whole new file with extra 35 lines but the current `app.py` has just 1 line removed? are there uncleaned statements?